src/lib.rs

```diff
@@ -98,7 +99,7 @@ pub fn create_tolerations() -> Vec<Toleration> {
     ]
 }
 
-pub fn object_to_owner_reference<K: Meta>(meta: ObjectMeta) -> OperatorResult<OwnerReference> {
+pub fn object_to_owner_reference<K: Resource>(meta: ObjectMeta) -> OperatorResult<OwnerReference> {
```

Comment:
I won't pretend to fully understand the difference, but could you just briefly comment on why the other Metas don't need to be substituted by "Resource"?

Reply:
The only thing we access from `K` are the associated consts `API_VERSION` and `KIND` and they are from `Resource`.

`Resource` is a supertrait for `Meta` so this works as well but we don't need to be as _tight_.
Every `Meta` is also a `Resource` but not every `Resource` does have to be `Meta`.